requirements.txt

```diff
@@ -1,2 +1,3 @@
 clickhouse-driver==0.1.2
 apache-airflow==1.10.6
+pandas
```

Comment:
Not sure that I should to set a version. I build DataFrame using [__setitem__](https://github.com/pandas-dev/pandas/blob/master/pandas/core/frame.py#L2878) is standard behaviour.

Reply:
`apache-airflow==1.10.6` depends on desired version of `pandas` itself. I guess there is no need to specify it.